buildSrc/lint.xml

```diff
@@ -43,5 +43,7 @@
     <!-- Disable all lint checks on transformed classes by default. -->
     <issue id="all">
         <ignore path="**/.transforms/**" />
+        <!-- playground builds have dependency files in "transformed" instead of ".transforms" -->
```

Comment:
note that this is actually merged here:
https://android-review.git.corp.google.com/c/platform/frameworks/support/+/2599045

but kept it in this CL so i PR here also passes (well, except for navigation which is referring an unreleased artifact)